src/styles/card.scss

```diff
@@ -0,0 +1,90 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  font-weight: 400;
+  width: 200px;
+  height: 408px;
+  border: 1px solid rgb(243, 243, 243);
+  border-radius: 5px;
+
+  &__image {
+    margin-top: 32px;
+    width: 160px;
+    height: 134px;
+  }
+
+  &__product-description {
+    width: 166px;
```

Comment:
it's better to add correct padding to the `.card` and set width `100%` (but in most cases you don't even need to set it since block elements have 100% width by default)

Reply:
still not fixed:
![image](https://github.com/mate-academy/layout_product-cards/assets/86923089/4f018ced-92ea-4927-beaf-2046cb3f32fa)

If you have a question related to this comment (or any other question) feel free to ask in the chat